### DISCUSSION TITLE
Update device

### DISCUSSION THREAD

**Discussion by tizfra** [GeekFunkLabs/fluidpatcher#43] [Q&A] [answered]
Hi,
on Squishbox in `system menu..` I find `Update device`, but it remain stuck on `checking`.
How can I update without lose any data?

**Comment by albedozero** [comment, marked as answer]
This was a weird bug - it seems like maybe github made a change to how their web pages redirect that broke some code. This should be fixed by my recent [commit](https://github.com/albedozero/fluidpatcher/commit/c42e83e11bbe376564e6e8e26efdf6742fcc3b93). To get the fix so the Update Device menu will work again, you'll have to do a `curl -L git.io/squishbox | bash` from the command line. You can say "no"/"no change" to everything except updating FluidPatcher and you should be good to go.
